libs/designsystem/accordion/src/accordion-item.component.spec.ts

```diff
@@ -72,4 +80,12 @@ describe('AccordionItemComponent', () => {
       });
     });
   });
+  it('should emit the toggle event when collapsed', () => {
```

Comment:
Consider code readability:
```suggestion

  it('should emit the toggle event when collapsed', () => {
```